src/main/target/KAKUTEF4/target.h

```diff
@@ -88,7 +88,7 @@
 #define I2C_DEVICE              (I2CDEV_1)
 
 #if defined(FLYWOOF405)
-#define I2C1_SCL                PB6
+#define I2C1_SCL                PB8//
```

Comment:
I think `//` shouldn't be there, let's remove it?
Also notice that original `KAKUTEF4` uses `PB8` as well, so this code should be stripped off conditionals now:
```diff
- #if defined(FLYWOOF405)
- #define I2C1_SCL PB6
- #else
- #define I2C1_SCL PB8
- #endif
+ #define I2C1_SCL PB8
```